src/lbaf/Model/lbsWorkModelBase.py

```diff
@@ -28,25 +28,24 @@ def factory(work_name, parameters, lgr: Logger):
         from .lbsAffineCombinationWorkModel import AffineCombinationWorkModel
 
         # Ensure that work name is valid
+        # TODO: not needed anymore, because of configuration schema cheking
         try:
             # Instantiate and return object
             work = locals()[work_name + "WorkModel"]
             return work(parameters, lgr=lgr)
         except:
             # Otherwise, error out
-            LGR.error(f"Could not create a work with name {work_name}")
-            sys.exit(1)
+            LGR.error(f"Could not create a work with name: {work_name}")
+            raise NameError(f"Could not create a work with name: {work_name}")
 
     @abc.abstractmethod
     def compute(self, rank):
         """ Return value of work for given rank
         """
         # Must be implemented by concrete subclass
-        pass
 
     @abc.abstractmethod
     def aggregate(self, values: dict):
         """ Return value of work given relevant dictionary of values
         """
         # Must be implemented by concrete subclass
-        pass
```

Comment:
No, abstract method - not instantiated, no need to pass anything. As you can see works without `pass`.

Reply:
I know it **works**.
But please note that this was not my question.

My question was about whether is is a Python **good practice** (style). I seemed to recall that it was.

Reply:
https://vald-phoenix.github.io/pylint-errors/plerr/errors/basic/W0107.html

Reply:
Nothing specific regarding `pass`.

In practice when there is docstring no pass is needed:
```Python3
    @abc.abstractmethod
    def compute(self, rank):
        """ Return value of work for given rank
        """
```
or when there is no docstring, pass must be used, because of `IndentationError`:
```Python3
    @abc.abstractmethod
    def compute(self, rank):
        pass
```
Both of the above are correct, the below causes `IndentationError` and is wrong:
```Python3
    @abc.abstractmethod
    def compute(self, rank):
```
So, please decide and I will apply the decision to the code.

Reply:
Sounds good to me. I thought it was some style rule. But if it's not then let's not bother with it.

Thanks for the research.